wintaser/GUI/DlgBase.cpp

```diff
@@ -88,7 +88,10 @@ namespace
                                                      (sizeof(WORD) * 1);
 };
 
-DlgBase::DlgBase(std::wstring caption, SHORT x, SHORT y, SHORT w, SHORT h)
+std::map<HWND, DlgBase*> DlgBase::ms_hwnd_to_dlgbase_map;
+
+DlgBase::DlgBase(std::wstring caption, SHORT x, SHORT y, SHORT w, SHORT h, bool tab_page) :

```

Comment:
Might be better to have tab_page as an enum, in case we want to add more options than just tabbed or regular.